helpers/release.md

```diff
@@ -0,0 +1,10 @@
+# Release process
+
+* Update the [changelog](/CHANGELOG.md)
+* Update the stack versions in [bumper.py](/helpers/bumper.py) and run the script. This will update the versions in all the right places
+* Open a pull request and wait for a green build before merging
+* Create a [new release](https://github.com/elastic/helm-charts/releases/new) and include the latest changelog entry
+* Run the [release script](/helpers/release.py) to build and upload the artifact
+  ```
+  GCS_BUCKET=elastic-helm-charts python release.py
```

Comment:
@Crazybus thanks for doing this doc. 
Could it possible to have a Docker container for running both `bumper.py` and `release.py` scripts?

Reply:
Sure. I have created an issue to track doing this https://github.com/elastic/helm-charts/issues/113. This pull request is about documenting the current process so I'm leaving it out of this one.

Reply:
Great, thanks 👍